units/patch_intent.go

```diff
@@ -349,25 +351,88 @@ func (j *patchIntentProcessor) buildCliPatchDoc(ctx context.Context, patchDoc *p
 			patchDoc.Githash, projectRef.Identifier)
 	}
 
+	summaries, err := getPatchSummaries(patchDoc)
+	if err != nil {
+		return errors.Wrap(err, "error getting patch summary")
+	}
+
+	patchDoc.Patches[0].ModuleName = ""
+	patchDoc.Patches[0].PatchSet.Summary = summaries
+
+	return nil
+}
+
+func getPatchSummaries(patchDoc *patch.Patch) ([]patch.Summary, error) {
 	var reader io.ReadCloser
-	reader, err = db.GetGridFile(patch.GridFSPrefix, patchDoc.Patches[0].PatchSet.PatchFileId)
+	reader, err := db.GetGridFile(patch.GridFSPrefix, patchDoc.Patches[0].PatchSet.PatchFileId)
 	if err != nil {
-		return err
+		return nil, errors.Wrap(err, "error getting patch file")
 	}
 	defer reader.Close()
-	bytes, err := ioutil.ReadAll(reader)
-	if err != nil {
-		return err
+	var patchBytes []byte
+	// if commit queue then mbox format, squash body to make summary
+	if patchDoc.Alias == evergreen.CommitQueueAlias {
+		patchBytes, err = handleFormattedPatch(reader)
+		if err != nil {
+			grip.Debug(message.WrapError(err, message.Fields{
+				"ticket": "problem parsing mbox",
+				"patch":  patchDoc.Id.Hex(),
+				"desc":   patchDoc.Description,
+				"alias":  patchDoc.Alias,
+			}))
+			return nil, errors.Wrap(err, "error parsing formatted patch")
+		}
+		// TODO: in the future this should just error
+		// refresh reader to try again in the case of an old commit queue item
+		reader, err = db.GetGridFile(patch.GridFSPrefix, patchDoc.Patches[0].PatchSet.PatchFileId)
+		if err != nil {
+			return nil, errors.Wrap(err, "error getting patch file")
+		}
 	}
+	if len(patchBytes) == 0 {
+		patchBytes, err = ioutil.ReadAll(reader)
+		if err != nil {
+			return nil, errors.Wrap(err, "error parsing patch")
+		}
+	}
+	return thirdparty.GetPatchSummaries(string(patchBytes))
```

Comment:
if all we're doing with the diff is getting summaries, how does the mbox package help us? Does `git apply  --numstat` not work on mbox formatted diffs?